packages/@sanity/cli/src/actions/init-project/determineAppTemplate.ts

```diff
@@ -1,4 +1,4 @@
-const appTemplates = ['app-quickstart']
+const appTemplates = ['app-quickstart', 'app-quickstart-sanity-ui']
```

Comment:
What about just `app-sanity-ui` adding quickstart makes it mouthful

```suggestion
const appTemplates = ['app-quickstart', 'app-sanity-ui']
```

Reply:
Sure, works for me! I'll update the other files